ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ultrassd-disk/provision/cucushift-installer-rehearse-azure-ipi-ultrassd-disk-provision-chain.yaml

```diff
@@ -0,0 +1,20 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-ultrassd-disk-provision
+  steps:
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: ipi-conf-azure
+  - ref: ipi-conf-azure-custom-region
+  - ref: ipi-conf-azure-custom-az
+  - ref: ipi-conf-azure-ultrassd
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  env:
+  - name: CUSTOM_AZURE_REGION
+    default: "centralus"
```

Comment:
Per my testing, even we set AZ to "1" in centralus, also always failed, so suggest to change it to `westus2`

Reply:
ok , let me make the change .